### ISSUE TITLE
existingSecret name is hard coded

### ISSUE THREAD

**Issue by drbothen** [bootc/netbox-chart#13]
It looks like the exsistingSecret was accidently hard coded to the name of the pod based on the error i was getting noty able to mount secret and the following lines in the deployment template:

 ```
- name: secrets
          secret:
            secretName: {{ include "netbox.fullname" . }}
```

**Comment by bootc** [comment]
Ugh sorry I dropped the ball on this. I've got a fix in the wings and am planning to push an updated chart in the next few hours.

**Comment by bootc** [comment]
This is fixed in chart version `2.1.0`, which includes NetBox 2.8.1 (but can support NetBox 2.7.11 or above).
